atomic_reactor/plugins/build_source_container.py

```diff
@@ -150,13 +150,7 @@ def run(self):
                           maven_source_data_dir)
             shutil.rmtree(maven_source_data_dir)
 
-        image_tar_path = self.export_image(image_output_dir)
+        self.export_image(image_output_dir)
 
         self.log.info('Will remove unpacked image directory: %s', image_output_dir)
         shutil.rmtree(image_output_dir)
-
-        return BuildResult(
```

Comment:
well we shouldn't return BuildResult anymore BUT, we should return the same as dict .....

Reply:
Hmm, and why do we want that?

Reply:
ahh right so we know that output dir already in advance, right so logs and skip_layer as return looks good